scripts/snapshot.mjs

```diff
@@ -85,10 +81,14 @@ for (const flags of flagCombinations) {
 }
 
 // Filter out combinations that are not allowed
-flagCombinations = flagCombinations.filter(
-  (combination) =>
-    !featureFlagsDenylist.some((denylist) => denylist.every((flag) => combination.includes(flag))),
-)
+flagCombinations = flagCombinations
+  .filter(
+    (combination) =>
+      !featureFlagsDenylist.some((denylist) =>
+        denylist.every((flag) => combination.includes(flag)),
+      ),
+  )
+  .filter((combination) => !(combination.length === 1 && combination[0] === 'bare'))
```

Comment:
nit: can you add a comment here to explain why this line?

Reply:
Added